### PR TITLE
feat: Release/v2.0.0

### PR DESCRIPTION
**Pull request by vsukhin**
## Checklist (choose whats happened)

- [ ] breaking change! (describe)
- [ ] tested locally
- [ ] tested on cluster
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

## Breaking changes

-

## Changes

-

## Fixes

-

### CODE REVIEW

File: charts/testkube-api/templates/_helpers.tpl [hunk 1]
```diff
@@ -163,8 +163,6 @@ Define API environment in agent mode
 - name: TESTKUBE_PRO_MIGRATE
   value:  "{{ .Values.cloud.migrate }}"
 {{- end}}
-- name: "NATS_EMBEDDED"
```
Comment (emil2k) [inline charts/testkube-api/templates/_helpers.tpl:166]: I cannot see the commit that removes this option to see the reasoning behind it.
Did this go straight to main?

File: charts/testkube-api/templates/testworkflows/builtin-templates/beta/artillery.yaml [hunk 2]
```diff
@@ -5,7 +5,6 @@ metadata:
   name: official--artillery--beta
   labels:
     testkube.io/name: Artillery
-    testkube.io/wizard: disabled
```
Comment (emil2k) [inline charts/testkube-api/templates/testworkflows/builtin-templates/beta/artillery.yaml:8]: @WitoDelnat Did you mean to remove this?

File: charts/testkube-api/templates/_helpers.tpl (hunk 1 above)
Comment (vsukhin) [reply charts/testkube-api/templates/_helpers.tpl:166]: I'm merging dev to main. Suppose, that dev should contain the latest version. But might be, it was added only to main. We can wait @exu to confirm

Comment (emil2k) [reply charts/testkube-api/templates/_helpers.tpl:166]: He is on vacation till the end of the week, but yes I think that is what happened.

Comment (vsukhin) [reply charts/testkube-api/templates/_helpers.tpl:166]: Let's wait then, might be the whole pr is just missed commits from main to dev

**Pull request closed without merging**